### PR TITLE
Corrects line-heights and spacing through the app

### PR DESCRIPTION
**Pull request by joaojusto**
The objective is to correct spacing and line-heights on the mobile version. No desktop concerns here

### CODE REVIEW

File: app/assets/javascripts/app/discussions/index.html.slim [hunk 1]
```diff
@@ -11,9 +11,9 @@ div ui-view="sidebar"
   .TopBarLayout-content
     .Page
       section.DiscussionList layout='column' layout-margin=''
-        md-container.u-upperCase flex='' Open Discussions
-        md-container layout='column' layout-margin='' wait-for='ctrl.pageReady'
+        md-container.DiscussionList-state flex='' Open Discussions
```
Comment (gabrielpoca) [inline app/assets/javascripts/app/discussions/index.html.slim]: Do you think this element represents the `state` of the `DiscussionList`? What about line 19?

**Comment by joaojusto** [comment]
@gabrielpoca bump

File: app/assets/stylesheets/components/_theme.css.scss [hunk 2]
```diff
@@ -86,7 +86,8 @@ $theme-font-family: "brandon-grotesque";
 
 /** Spacing **/
 $default-spacing: 1rem;
-
+$default-padding: 1rem;
```
Comment (gabrielpoca) [inline app/assets/stylesheets/components/_theme.css.scss]: there is no such things as a `$default-padding` and a `$default-card-padding`, you don't need these

File: app/assets/stylesheets/components/_discussion.css.sass [hunk 3]
```diff
@@ -8,7 +8,7 @@ $discussion-item-triangle-size: 40px
   background-color: theme-palette(neutral, white)
   cursor: pointer
   padding-right: $discussion-item-triangle-size
-
+  margin-top: 0
```
Comment (gabrielpoca) [inline app/assets/stylesheets/components/_discussion.css.sass]: Why is this necessary now? It was working before

**Comment by gabrielpoca** [comment]
:+1:
